nemo_skills/evaluation/aggregate_answers.py

```diff
@@ -195,13 +195,15 @@ def _read_predictions(self) -> Tuple[List, List]:
         cfg = self.cfg
         new_answers = []
         all_predictions = []
-        for idx, predictions in enumerate(tqdm(zip_longest(*self.input_file_handles))):
+        for idx, predictions in enumerate(tqdm(zip_longest(*self.input_file_handles, fillvalue="{}"))):
```

Comment:
are these changes necessary for your pipeline? E.g. is it expected to get different number of lines for different generations? I'd rather remove zip longest and fail with error actually, at least by default as skipping lines might lead to unexpected results if order is messed up